P5/Source/Guidelines/en/ND-NamesDates.xml

```diff
@@ -2269,6 +2096,254 @@ occupation and residence.</p>
+        may appear as content of a <gi>place</gi>, <gi>person</gi>, or <gi>org</gi>. However, it is also possible to describe events
+        independently of either a place, person, or organization. This may be useful not only for encoding texts which describe named events and 
+        their <gi>eventName</gi>s, but also in applications as chronologies or lists of significant events (e.g., important  battles, legislative milestones, or the stages of construction of a large building, etc.).</p>
+      
+      <p>The <gi>listEvent</gi> element is a member of the <ident
+        type="class">model.listLike</ident> class, and may therefore
+        appear inside <gi>standOff</gi>, or wherever else lists are
+        permitted, in the same way as the <gi>listPerson</gi>,
+        <gi>listPlace</gi>, <gi>listOrg</gi> etc. elements described elsewhere in this
+        chapter.</p>
+      <egXML xmlns="http://www.tei-c.org/ns/Examples">
+        <standOff>
+          <listEvent>
+            <event when="1713" ref="https://www.canadiana.ca/view/oocihm.9_01832/2">
+              <label>Treaty of Utrecht</label>
+              <!-- The following example presupposes that the editors of the TEI document decided
+                to understand legislation or a treaty as an <gi>event</gi>, which itself may be
+                considered a shortcut for several events that led up to the existence or validity
+                of the law or treaty (e.g. the signing of a treaty could also be modeled as an event,
+                and the result on paper could be encoded as an <gi>object</gi> or <gi>bibl</gi>). -->
+              <desc>France ceded to Great Britain its claims to the <orgName>Hudson's Bay Company</orgName> territories in
+                <placeName>Rupert's Land</placeName>, <placeName>Newfoundland</placeName>, and <placeName>Acadia</placeName> and
+                recognized British suzerainty over <orgName type="tribe">the Iroquois</orgName> but retained its other pre-war
+                North American possessions, including 
+                <placeName key="PEI">Île-Saint-Jean</placeName> 
+                (now <placeName key="PEI">Prince Edward Island</placeName>)...</desc>
+            </event>
+            <event when="1774" key="14-GeoIII-c83">
+              <label>Quebec Act</label>
+              <desc>This act of the British Parliament guaranteed free practice of the Catholic faith and restored use of the
+                French Civil Code for private matters throughout the Province of Quebec, which had been expanded in territory
+                following the <ref>Treaty of Paris</ref>.</desc>
+            </event>
+            <event when="1778" ref="http://avalon.law.yale.edu/18th_century/del1778.asp">
+              <label>Treaty of Fort Pitt</label>
+              <desc>Also known as the <name type="event">Treaty with the Delawares</name>, this was the first written treaty
```

Comment:
Okay--I'm reviewing our discussion of this from Paderborn, and I remember Council found this particular example to be problematic.  
We wanted to go through and revise the examples (this one in particular) to change `<label>` to `<eventName>`. 

At Paderborn we recommended that `<label>` be applied to a change of state, while `<eventName>` be used for an event with a name. (Admittedly changes in state like a coronation would probably be given a name, so this may be a little ambiguous.) But in this example, at this particular point, this is the only time we see `<name type="event">` while everything else is encoded with `<label>`. I think we did intend to change this example to match the new `<eventName>` or at least to make the encoding more consistent throughout (as in, apply `<label>` to "Treaty with the Delawares" and elevate it to the same position as "Treaty of Fort Pitt". @joeytakeda

Reply:
I see that we *also* recommended simply merging the PR to make `<eventName>` available at last, and open a ticket to improve the examples later, so I'm not going to raise this as an objection--just noting it as something we need to revisit! As soon as we merge this PR, we should open a new ticket. (I'll probably just do that.)  @joeytakeda